specs/product-spec.rb

```diff
@@ -0,0 +1,2 @@
+require_relative './spec-helper'
+require_relative '../lib/farmar_product'
```

Comment:
Because you're requiring the module in `spec_helper`, you likely don't need these lines. We should let the module be in charge of bringing everything together.